accepted/future-releases/primary-constructors/feature-specification.md

```diff
@@ -575,64 +575,81 @@ constructors as well.
 ```ebnf
 <classDeclaration> ::= // First alternative modified.
      (<classModifiers> | <mixinClassModifiers>)
-     'class' <classNamePart> <superclass>? <interfaces>? <classBody>
+     'class' <classNameMaybePrimary> <superclass>? <interfaces>? <classBody>
```

Comment:
Suggestion/nit: "maybe" implies "optional" to me but not a choice from two alternatives. Perhaps a better name is "classNameOrPrimary"?

Reply:
It is definitely determining the class name (so it's always `className`), and then it may also include a primary constructor (so that primary constructor is optional). So it's not a class name _or_ a primary constructor. I don't know if there's a better way to indicate this "definitely class name and possibly primary constructor" relationship.. I'll leave this thread open and keep the non-terminal for now.